miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleClassService.java

```diff
@@ -113,4 +114,25 @@ public SampleClass inferStockFromAliquot(SampleClass sampleClass) {
     return stockClass;
   }
 
+  @Override
+  public SampleClass inferTissueFromStock(SampleClass sampleClass) {
```

Comment:
With current OICR SampleClasses, there's only one Tissue class. With our previous Tissue classes, this would always fail because multiple tissue classes were valid for every Stock class. I'm not sure it's worth the DB lookups to pretend that you could just toss any classes into the DB and expect it to work. We could just treat it like Identity instead (expect there to only be one class in the category)?